pk_tic_tac_toe.rb

```diff
@@ -83,42 +83,31 @@ def check_win(board, player, victory)
   end
 end
 
-def play_turn(player, board, victory)
-  print "#{player.name}, it's your turn; "
+def play_turn(player, board)
   make_move(board, player)
   display_board(board)
-  check_win(board, player, victory) == true
+  check_win(board, player) == true
 end
 
-def play_game(board, player1, player2)
-  victory = [
-    [0, 1, 2], # same gp top row.
-    [3, 4, 5], # same gp middle row.
-    [6, 7, 8], # same gp bottom row.
-    [0, 3, 6], # same gp left column.
-    [1, 4, 7], # same gp center column.
-    [2, 5, 8], # same gp right column.
-    [0, 4, 8], # same gp left diagonal.
-    [6, 4, 2]  # same gp right diagnal.
-  ]
-  player1 = create_player1(player1)
-  player2 = create_player2(player2)
+def play_game(board)
```

Comment:
I now assign player (1 and 2) and game piece ("X" and "O") as arguments to the `create_player` method `player_num` and `game_piece` parameters (`create_player(player_num, game_piece`), and then assign these each to the `player1` and `player2` variables.